tests/JustEat.StatsD.Tests/StatsDPublisherTests.cs

```diff
@@ -90,9 +90,9 @@ public static void Metrics_Not_Sent_If_Array_Is_Null_Or_Empty()
             var config = new StatsDConfiguration();
             var anyValidTags = new Dictionary<string, string?>
             {
-                ["foo"] = "bar",
```

Comment:
I prefer the new dictionary syntax as it's a bit terser.

Reply:
Done!